cleanup/check_account.py

```diff
@@ -169,6 +193,9 @@ def verify_accounts(saa_api_key, saa_url):
             previous_cost = float(
                 previous_usage['Items'][0]['billable_cost']['N'])
 
+            push_to_prometheus(push_gw_url, 'verify_accounts', 'ibm_current_usage', 'Current usage by account',
+                               current_cost, labels)
```

Comment:
Why do we want to collect the `current_cost`? Without the previous cost, it won't matter in the dashboard. Wouldn't we want to calculate the delta and send that instead?

Reply:
I'm going to add the previous value as a new metric `ibm_previous_usage` and I can calculate the delta in the dashboard